lib/hacluster.pm

```diff
@@ -1159,7 +1187,13 @@ sub setup_sbd_delay() {
 }
 
 =head2 set_sbd_service_timeout
-    set_sbd_service_timeout($service_timeout)
+
+  set_sbd_service_timeout($service_timeout)
+
+Set the service timeout for the SBD service in the SUT to the number of
```

Comment:
```suggestion
Set the service timeout for the SBD service in the SUT to the number of
seconds passed as argument. 
This is obtained by configuring a systemd override file for the
SBD service.
```

Reply:
`obtained` is wrong there. Will replace it with `accomplished`.